identity-apps-core/apps/authentication-portal/src/main/webapp/fido2-auth.jsp

```diff
@@ -244,7 +244,7 @@
     <script type="text/javascript" src="libs/base64js/base64js-1.3.0.min.js"></script>
     <script type="text/javascript" src="libs/base64url.js"></script>
 
-    <% String clientId=request.getParameter("client_id"); %>
+    <% String clientId=Encode.forUriComponent(request.getParameter("client_id")); %>
```

Comment:
We should use `encode.forjavascriptblock` here since clientid is not used as a Url query param